### ISSUE TITLE
Slab implementation culls faces regardless of transparency

### ISSUE THREAD

**Issue by ghost** [Juuxel/VanillaParts#8]
It appears that multipart containers cull faces regardless of whether or not the block adjacent to them is transparent or not. This is most apparent if you place glass next to a multipart container as in the example below.

![Multipart Cullface Issue](https://user-images.githubusercontent.com/18454093/72764035-8ed12280-3ba3-11ea-8126-fa436d11e6cb.png)

Also seen with a torch on a fence post because for whatever reason it appears fences are automatically turned into multipart containers even when they don't need to be. 

![Torch On Fence](https://user-images.githubusercontent.com/18454093/72764231-6f86c500-3ba4-11ea-8c09-453e8092a839.png)

**Comment by Loriborn** [comment]
I'd like to add that because fences are all considered multipart containers even when they don't have to be, this breaks expected behavior with tool breaking and with attaching leads to fences. Animals on leads cannot be attached to any fences with Vanilla Parts enabled, and additionally, fences do not require the proper tool in order to break them effectively/faster. Even moreso, particles for breaking all multipart blocks, including isolated fences, have a missing texture and thus the particles default to the pink/black checkerboard.

**Comment by Juuxel** [comment]
> I'd like to add that because fences are all considered multipart containers even when they don't have to be

They aren't. Vanilla blocks are vanilla blocks when a player hasn't converted them to multiparts.

> Animals on leads cannot be attached to any fences with Vanilla Parts enabled

I'll open a separate issue for this.

> fences do not require the proper tool in order to break them effectively/faster

AFAIK, this is still unimplemented in LibMultipart. (cc @AlexIIL)

> Even moreso, particles for breaking all multipart blocks, including isolated fences, have a missing texture and thus the particles default to the pink/black checkerboard.

This is also a LibMultipart bug/unimplemented feature.

**Comment by AlexIIL** [comment]
> > fences do not require the proper tool in order to break them effectively/faster
>
> AFAIK, this is still unimplemented in LibMultipart. (cc @AlexIIL)

Yep, I created https://github.com/AlexIIL/LibMultiPart/issues/16 and https://github.com/AlexIIL/LibMultiPart/issues/17 for this.

> > Even moreso, particles for breaking all multipart blocks, including isolated fences, have a missing texture and thus the particles default to the pink/black checkerboard.
>
> This is also a LibMultipart bug/unimplemented feature.

Block broken particles have been implemented, however I'm still missing break*ing* particles. (https://github.com/AlexIIL/LibMultiPart/issues/1)
